code/modules/projectiles/updated_projectiles/guns/specialist.dm

```diff
@@ -741,8 +741,14 @@
 	wield_penalty = 1.6 SECONDS
 	aim_slowdown = 1.75
 	attachable_allowed = list(
-						/obj/item/attachable/magnetic_harness,
-						/obj/item/attachable/scope/mini)
+		/obj/item/attachable/bipod,
```

Comment:
Revert these attachment changes. They are noob traps for the most part, or offscreen damage on the main scope.